File: engine/model/src/main/scala/tools/packet/character/interaction/GetMiniRoomError.scala
package tools.packet.character.interaction

import net.opcodes.SendOpcode
import tools.packet.{MiniRoomError, PacketInput}

class GetMiniRoomError(private var _status: MiniRoomError) extends PacketInput {
  def status: MiniRoomError = _status

  override def opcode(): SendOpcode = SendOpcode.PLAYER_INTERACTION
}

File: engine/engine-model/src/main/java/tools/packet/character/interaction/GetMiniRoomError.java
package tools.packet.character.interaction;

import net.opcodes.SendOpcode;
import tools.packet.MiniRoomError;
import tools.packet.PacketInput;

public record GetMiniRoomError(MiniRoomError status) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.PLAYER_INTERACTION;
   }
}